'npm start' fails after fresh git clone

The starter does not start after a fresh clone:
```
git clone --depth 1 https://github.com/vaadin/base-starter-connect
cd base-starter-connect
npm install
npm start
```
the result is
```
$ npm start

> base-starter-connect@0.5.0 start /Users/viktor/IdeaProjects/vaadin-services/jens-starter
> webpack --mode development --silent && npm run start:backend -- -- npm run start:apibrowser -- -- npm run start:webpack

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! base-starter-connect@0.5.0 start: `webpack --mode development --silent && npm run start:backend -- -- npm run start:apibrowser -- -- npm run start:webpack`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the base-starter-connect@0.5.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/viktor/.npm/_logs/2019-01-22T12_10_45_736Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -18,7 +18,7 @@
     "start:backend": "node scripts/start/backend.js",
     "start:webpack": "webpack --mode development -w",
     "start:apibrowser": "node scripts/start/apibrowser.js",
-    "start": "webpack --mode development --silent && npm run start:backend -- -- npm run start:apibrowser -- -- npm run start:webpack",
+    "start": "npm run start:backend -- -- npm run start:apibrowser -- -- npm run start:webpack",
     "test:unit": "node $NODE_DEBUG_OPTION ./node_modules/.bin/intern environments=node functionalSuites=",
     "test:e2e": "node $NODE_DEBUG_OPTION ./node_modules/.bin/intern",
     "test": "npm run test:unit && npm run build:frontend && npm run start:backend -- --nowatch -- npm run test:e2e"

--- a/src/main/resources/META-INF/resources/index.html
+++ b/src/main/resources/META-INF/resources/index.html
@@ -1,0 +1,3 @@
+
+<h1>Did you run webpack?</h1>
+<p>The content of this file should be replaced with your application when you run webpack builder.</p>

